pushapkscript/docker.d/worker.yml

```diff
@@ -97,21 +102,34 @@ products:
           skip_check_same_locales: true
           skip_checks_fennec: true
           override_channel_model: "single_google_app"
-          single_app_config:
+          app:
             service_account: { "$eval": "GOOGLE_PLAY_SERVICE_ACCOUNT_REFERENCE_BROWSER" }
             credentials_file: { "$eval": "GOOGLE_CREDENTIALS_REFERENCE_BROWSER_PATH" }
             package_names:
               - "org.mozilla.reference.browser"
             certificate_alias: "reference-browser"
-
+        - product_names: [ "firefox-tv" ]
+          skip_check_multiple_locales: true
+          skip_check_same_locales: true
+          skip_checks_fennec: true
+          skip_check_signature: true
+          skip_check_ordered_version_codes: true
+          override_channel_model: "single_google_app"
+          apps:
+            production:
+              package_names:
+                - "org.mozilla.tv.firefox"
+              amazon:
+                client_id: 'dummy'
```

Comment:
I'll amend later this week these to reflect the correct env var once I sent it out to Jeremy and update cloudops-infra repo too.

Reply:
Just to clarify - we're not deploying this to `firefox-tv` repo until we add these missing credentials.